coverage_assessment.R

```diff
@@ -41,9 +41,131 @@ nut <- get_data()
 
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
+####MAM
 
+with(nut, sum(muac >= 11.5 & muac <12.5  & age >=6 & age <60 & oedema == 2, na.rm= TRUE))
 
+##number of moderate acute malnourished (MAM) cases children 6-59 months old 
 
+Male_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <60 & oedema == 2 & sex == 1 , na.rm= TRUE ))
+
+##assigning moderate acute malnourished (MAM) male 6-59 months old to Male_MAM
+
+Female_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <60 & oedema == 2 & sex == 2 , na.rm= TRUE ))
+
+##assigning  moderate acute malnourished (MAM) female 6-59 months old to Female_MAM
+
+UM_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 & age >=6 & age <60 & oedema == 2 & county == "Urban Montserrado" , na.rm= TRUE))
+
+##assigning moderate acute malnourished (MAM) cases children 6-59 months old  in Urban Montserrado to UM_MAM
+
+GB_MAM <- with(nut, sum(muac >= 11.5 & muac <12.5 &age >=6 & age <60 & oedema == 2 & county == "Grand Bassa" , na.rm= TRUE))
+
+##assigning  moderate acute malnourished (MAM) cases children 6-59 months old other than Urban Montserrado to GB_MAM
+
+MAM <- data.frame(Male_MAM, Female_MAM, UM_MAM, GB_MAM)
+
+## creating a data frame for MAM
+
+if(interactive()) View(MAM)
+
+##Viewing MAM data
+
+
+
+####Alternate table method for number of moderate,severe and normal malnutrition by sex and by location:
+
+num_data <- with(nut, data.frame(sex,county,muac,oedema))
+
+##assigning the data frame to num_data
+
+SEX <- ifelse(num_data$sex == "1", "Male", "Female")
+OEDEMA <- ifelse(num_data$oedema == "1", "Yes", "No")
+MUAC <- with( num_data, ifelse (muac <11.5, "Severe",
+                               ifelse(oedema == 1, "Severe",
+                                      ifelse(muac >=11.5 & muac<12.5, "Moderate",
+                                             ifelse (muac >=12.5, "Normal", "No")))))
+
+list <- data.frame(SEX,MUAC, OEDEMA, num_data$county)
```

Comment:
`list` is a name of a function in R (see `?list()`). You should ideally avoid creating objects with names the same as functions in R. So, better call this something else like `new_nut_data` or something like that.

Or, to avoid having to create a new object for the data, you can add the new variables you created to the `nut` dataset as shown below.